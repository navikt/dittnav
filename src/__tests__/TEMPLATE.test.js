import * as React from 'react';
const ReactTestRenderer = require('react-test-renderer');

test('basic green component snaphot-test', () => {
  const component = ReactTestRenderer.create((<div>Some very fancy component</div>));
  expect(component.toJSON()).toMatchSnapshot();
});
