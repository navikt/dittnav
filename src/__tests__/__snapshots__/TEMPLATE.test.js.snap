// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic green component snaphot-test 1`] = `
<div>
  Some very fancy component
</div>
`;
